components/list_item_group/list_item_group.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <ul
+    :id="id"
+    :class="['d-ps-relative', 'd-px0', listClass]"
+    data-qa="dt-dropdown-list-wrapper"
+    :aria-labelledby="`${id}-heading`"
+  >
+    <li
+      v-if="heading"
+      :id="`${id}-heading`"
+      role="presentation"
+      data-qa="dt-dropdown-list-heading"
+      :class="['dt-dropdown-list--header', headingClass]"
```

Comment:
Will `dt-dropdown-list--header` provide some default styling? The search [figma for DP 2.0](https://www.figma.com/file/ymr1mdTU5RMU8UTy7IIYKZ/Search-Improvements-7%2F30?node-id=4340%3A229431) has some styling for its heading, but I think it's up to the team to decide how the default should look like

Reply:
Yeah I think we will just leave it unstyled for now. I'll check with design and see if they have some ideas for this though.